SpatialGDK/Source/SpatialGDK/Private/Interop/WellKnownEntitySystem.cpp

```diff
@@ -52,6 +57,17 @@ void WellKnownEntitySystem::Advance()
 	}
 }
 
+void WellKnownEntitySystem::SendClaimPartitionRequest(Worker_EntityId SystemWorkerEntityId, Worker_PartitionId PartitionId) const
```

Comment:
What was the reasoning behind putting this logic in the well known entity system? Seems more NetDriver level. For instance on startup flow a server will create it's entity, claim it, and then use that interest to "complete" the well known entity system.

Reply:
Reasoning: Partitions are counted as well known entities for this system (potentially they shouldn't be. The definition of a well known entity is too vague, which is my bad, needs revisiting). A goal is to move all uses of commands to the various subview-based systems (last thing until receiver and sender can be deleted) and so I thought partition commands would make sense to be done in the well known entity system, as the well known entity system will probably transform into the "startup checkpoint system" as of https://improbableio.atlassian.net/browse/UNR-4849.

Maybe it would be better to have an explicit partition system which handles this kind of thing and outputs a signal for the well known entity system to know when it's time to be "completed". Not sure.

Reply:
Wait are partitions even counted as well known entities, I think I'm totally wrong

which system is responsible for partitions?

Reply:
Partitions can be many entity types currently, player controllers, worker entities, generic partition entities. The only well known partition entity is the one stored in the snapshot.

Reply:
Trying to think how this could be incorporated into a single system, and I'm not sure it's necessary. It's more that partitions are used across multiple systems, and the commands required to orchestrate them could come from multiple sources.

Reply:
I agree- I think partitions should likely be handled by whatever system cares about that partition. This will be gone as of strategy worker anyway though, so I am leaning towards this not being too important for the minute